### PR TITLE
Satishkumaragarwal/custom render coin issue

### LINKED ISSUE #5033: Persona component onRenderCoin property not working
### Bug Report

- __Package version(s)__: Current
- __Browser and OS versions__: 

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? Yes

Requested priority: Normal

Products/sites affected: Calendar/Mail

#### Describe the issue:

Persona component provides **onRenderCoin** property to override the default initials coin. But when a custom render function is passed, the component still displays the initial coin.

#### Actual behavior:

The initials coin renders above the custom coin.

#### Expected behavior:

Component should render only the custom coin.

### PR DESCRIPTION
**Pull request by satishkumaragarwal**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #5033 
- [ ] Include a change request file using `$ npm run change`

#### Description of changes

Added a check to avoid rendering imageurl if render coin callback is present.

#### Focus areas to test

Persona card

### CODE REVIEW

File: common/changes/office-ui-fabric-react/satishkumaragarwal-customRenderCoinIssue_2018-06-20-16-20.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Fixing issue with render coin on persona component",
+      "type": "patch"
```
Comment (micahgodbolt) [inline common/changes/office-ui-fabric-react/satishkumaragarwal-customRenderCoinIssue_2018-06-20-16-20.json]: only commit this change file. remove the others with 'type': "none"

Comment (satishkumaragarwal) [reply common/changes/office-ui-fabric-react/satishkumaragarwal-customRenderCoinIssue_2018-06-20-16-20.json]: Done. Thanks.

Comment (micahgodbolt) [reply common/changes/office-ui-fabric-react/satishkumaragarwal-customRenderCoinIssue_2018-06-20-16-20.json]: Sorry, I should have said this before. But please update the "comment" above to describe the actual bug. This text actually gets used in our release notes, so knowing generally what the bug was is really helpful.

**Comment by cliffkoh** [comment]
@Markionium could you please review this when you get the chance?

File: packages/office-ui-fabric-react/src/components/Persona/PersonaCoin/PersonaCoin.base.tsx [hunk 2]
```diff
@@ -107,31 +107,31 @@ export class PersonaCoinBase extends BaseComponent<IPersonaCoinProps, IPersonaSt
...
-                  )}
-                  style={coinSizeStyle}
-                  aria-hidden="true"
-                >
-                  {onRenderInitials(this.props, this._onRenderInitials)}
-                </div>
+          size !== PersonaSize.size10 && size !== PersonaSize.tiny ? (
+            <div {...coinProps} className={classNames.imageArea} style={coinSizeStyle}>
+              {onRenderCoin === this._onRenderCoin && !this.state.isImageLoaded &&
+                (!imageUrl || this.state.isImageError) && (
```
Comment (Markionium) [inline packages/office-ui-fabric-react/src/components/Persona/PersonaCoin/PersonaCoin.base.tsx]: Could we pull this logic out into a descriptive `const`? It's becomes quite a difficult piece to understand by itself like this

Comment (Markionium) [inline packages/office-ui-fabric-react/src/components/Persona/PersonaCoin/PersonaCoin.base.tsx]: I also think we have unit tests for this. Could you add a test that renders a custom one and asserts it renders correctly and that the original one does not render?

**Comment by dzearing** [comment]
@satishkumaragarwal Would you mind resolving conflicts and address the feedback? The PR has been sitting for a while. Thank you!

**Comment by satishkumaragarwal** [comment]
@dzearing Sure will do.

**Comment by dzearing** [comment]
@satishkumaragarwal Once you've addressed issues @Markionium can help get them merged.

File: packages/office-ui-fabric-react/src/components/Persona/PersonaCoin/PersonaCoin.base.tsx (hunk 2 above)
Comment (satishkumaragarwal) [reply packages/office-ui-fabric-react/src/components/Persona/PersonaCoin/PersonaCoin.base.tsx]: Yes I updated the CustomCoinRender example test case in this PR. I believe that's the test case for this scenario?

**Pull request merged**